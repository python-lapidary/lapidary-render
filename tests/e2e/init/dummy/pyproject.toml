[build-system]
build-backend = "poetry.core.masonry.api"
requires = ["poetry-core>=1.3.2"]

[tool.poetry]
name = "dummy"
description = "Client library for dummy schema"
version = "0.1.0"
authors = []
license = ""
packages = [
    { include = 'test_dummy', from = "gen" },
]

[tool.poetry.dependencies]
python = "^3.9"
lapidary = {path="../../../.."}

[tool.poetry.group.dev.dependencies]
lapidary-render = "^0.9.0"

[tool.lapidary]
document_path = "dummy.yaml"
package = "test_dummy"
