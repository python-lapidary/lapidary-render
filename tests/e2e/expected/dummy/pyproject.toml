[build-system]
build-backend = "poetry.core.masonry.api"
requires = ["poetry-core>=1.3.2"]

[tool.poetry]
name = "dummy"
description = "Client library for dummy schema"
version = "0.1.0"
authors = []
license = ""
packages = [
    { include = 'test_dummy', from = "gen" },
]

[tool.poetry.dependencies]
python = "^3.9"
lapidary = {path="../../../../../lapidary"}

[tool.poetry.group.dev.dependencies]
lapidary-render = {path="../../../..", python="^3.12"}

[tool.lapidary]
document_path = "dummy.yaml"
package = "test_dummy"
